### PR TITLE
[OBSDEF-15341] Toggle CSS changes

### PR DESCRIPTION
**Pull request by singha158ecs**
This PR contains dell design wrapper changes for toggle.

toggle css

### CODE REVIEW

File: src/styles/components/Toggle.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            }
+        }
+
+        &:disabled{
+            & + label::before{
+                background-color: $gray-450 !important;
+            }
+            & + label:hover::before{
+                background-color: $gray-450 !important;
+            }
```
Comment (akate1) [inline src/styles/components/Toggle.scss:48]: Check if below code change works
```suggestion
            & + label::before,  & + label:hover::before {
                background-color: $gray-450 !important;
            }
       ```

File: src/styles/components/Toggle.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+@import "../common/colors";
+
+.clr-toggle-wrapper {
+    margin-right: 8px;
+    margin-bottom: 16px;
```
Comment (akate1) [inline src/styles/components/Toggle.scss:15]: can we use rem for margin as well ?

Comment (singha158ecs) [reply src/styles/components/Toggle.scss]: Done.

File: src/styles/components/Toggle.scss (hunk 1 above)
Comment (singha158ecs) [reply src/styles/components/Toggle.scss]: Done.

**Review by akate1 — APPROVED** [review]

**Review by psivakumaran — APPROVED** [review]

**Pull request merged**